scripts/prepare_sector_network.py

```diff
@@ -2944,83 +2942,91 @@ def add_heat(
         )
 
         if options["tes"]:
-            n.add("Carrier", f"{heat_system} water tanks")
+            if options["district_heating"]["ttes"]["enable"]:
```

Comment:
It would be nicer if indentation level could be reduced by one level for this section.